### ISSUE TITLE
arena 1-3 Target of target great inclusion for healers.

### ISSUE THREAD

**Issue by Tuckyb** [SweepyBoop/Sweepy-Boop#278]
![arena 1-3](https://github.com/user-attachments/assets/fcc530c4-90d3-4d4c-8793-32643a7301cf)

Hey would be amazing if the addon had target of target for arena 1-3 this is amazing for healers as it allows you to see party member HP on the arena frames but also who is being targeted with burst a lot of healers are currently using gladius just got get this functionality and moving the settings to make it work on their Sarena. This would be a great include

**Comment by SweepyBoop** [comment]
If you are requesting this in order too see which party member needs heal, I think the existing module does a much better job:
![image](https://github.com/user-attachments/assets/c96bef22-e438-457a-a631-19baab581983)

it shows yellow aggro border when a party member is targeted by one enemy player, and red border if targeted by at least 2 players. 

With the feature you request you need to check arena frames first then translate the target info to which raid frame you need to target and heal, and it's confusing when you have 2 teammates of the same class (very possible in solo shuffles)

**Comment by SweepyBoop** [comment]
![PvP raid frame aggro highlight](https://github.com/user-attachments/assets/6b985a1d-0c4d-4881-9659-c47ebb34c280)

**Comment by SweepyBoop** [comment]
This from Dannycarryx looks pretty neat:
![Image](https://github.com/user-attachments/assets/ced908cb-06b6-46d5-99f6-e20feda709c7)
It shows class colored bars on raid frames to indicate which enemy classes are targeting each teammate.
I'll look to implement a similar feature

**Comment by Tuckyb** [comment]
Yeah I actually meant on the arena frames like this. One of the best usecases is when you are healing and people are rotting being able to see who peple are actually targeting becomes very strong because you can allocate healing priority better.
    On Saturday 15 February 2025 at 05:25:47 am AEDT, SweepyBoop ***@***.***> wrote:  
 
  
This from Dannycarryx looks pretty neat:
image.png (view on web)
It shows class colored bars on raid frames to indicate which enemy classes are targeting each teammate.
I'll look to implement a similar feature

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
SweepyBoop left a comment (SweepyBoop/Sweepy-Boop#278)
This from Dannycarryx looks pretty neat:
image.png (view on web)
It shows class colored bars on raid frames to indicate which enemy classes are targeting each teammate.
I'll look to implement a similar feature

**Comment by SweepyBoop** [comment]
https://x.com/sweepyboop/status/2081474905259020437?s=20
